src/hotspot/share/cds/filemap.hpp

```diff
@@ -117,7 +117,7 @@ class FileMapHeader: private CDSFileMapHeaderBase {
   bool    _compressed_oops;                       // save the flag UseCompressedOops
   bool    _compressed_class_ptrs;                 // save the flag UseCompressedClassPointers
   int     _narrow_klass_pointer_bits;             // save number of bits in narrowKlass
-  int     _narrow_klass_shift;                    // save shift width used to pre-compute narrowKlass IDs in archived heap objects
+  int     _narrow_klass_shift;                    // save shift width used to pre-compute narrowKlass IDs in archived heap objectsa
```

Comment:
```suggestion
  int     _narrow_klass_shift;                    // save shift width used to pre-compute narrowKlass IDs in archived heap objects
```

Minor typo, don't bother fixing if it will result in a re-review cycle